state/forkid.go

```diff
@@ -17,6 +23,37 @@ func (s *State) UpdateForkIDIntervals(intervals []ForkIDInterval) {
 	s.cfg.ForkIDIntervals = intervals
```

Comment:
To avoid misunderstanding I suggest to rename this function to something like UpdateForkIdIntervalsStruct, as the current name seems that is updating the DB